spec/factories/user_factory.rb

```diff
@@ -57,6 +57,18 @@
       end
     end
 
+    trait :with_roomLimit_100_permission do
+      after(:create) do |user|
+        create(:role_permission, role: user.role, permission: create(:permission, name: 'RoomLimit'), value: '100')
+      end
+    end
```

Comment:
I wouldn't make this a trait either since we would expect a user to be able to create a number of rooms. Do it in the `after(:create)` on line 14, similar to `CreateRoom`